.github/CONTRIBUTOR_GUIDE/json/AppName.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "AppName",
+    "slug": "appname",
+    "categories": [
+        0
+    ],
+    "date_created": "DATE CREATED",
+    "type": "ct",
+    "updateable": true,
+    "privileged": false,
+    "interface_port": DEFAULT-PORT,
```

Comment:
maybe use something representative for this example, either 80 or 8080

Reply:
Can do.
But you can also change in the branch as you like.